tensorrt_llm/_torch/modules/fused_moe/communication/nccl_ep.py

```
@@ -335,11 +348,12 @@ def combine(
                     f"combine input shape={tuple(final_hidden_states.shape)} "
                     f"expected={expected_shape}"
                 )
-            final_hidden_states = final_hidden_states.view(
-                self.ep_size,
-                self.max_tokens_per_rank,
-                self.hidden_size,
-            )
+            if not os.environ.get("TRTLLM_NCCL_EP_ALGO", "LOW_LATENCY").upper() in ("HIGH_THROUGHPUT", "HT"):
```

Ruff (0.16.1)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)